shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -282,13 +284,14 @@ \subsection{New Epoch Transition}
   %
   \emph{Abstract pseudorandom schedule function}
   \begin{align*}
-    & \fun{overlaySchedule} \in \Seed \to \unitInterval \to \unitInterval \to
-      (\Slot\mapsto\VKeyGen^?) \\
+    & \fun{overlaySchedule} \in \powerset{\VKeyGen} \to \Seed \to \unitInterval \to
+        \unitInterval \to (\Slot\mapsto\VKeyGen^?) \\
   \end{align*}
   %
   \emph{Constraints}
   \begin{align*}
-    \text{ given: }~\var{osched}\leteq\fun{overlaySchedule}~\eta~\var{d}~\var{f} \\
+    \text{ given: }~\var{osched}\leteq\overlaySchedule{gkeys}{\eta}{d}{f} \\
```

Comment:
would it be clearer if `overlaySchedule` took the protocol parameters instead of `d` and `f`, so that in  the constraint you could see which parameters are being used? or just add a parenthetical to the "Constraints" title? I'm just wondering how to fig this info into the table...

Reply:
I think it would be best to add ` d := \fun{d}{pp}` and `f` analogously. As both are used for the other constraints, too.

Reply:
makes sense! I think it looks nice just taking the proto params and then using the accessors in the constraint.